docs/HISTORY.md

```diff
@@ -8,6 +8,12 @@ Around 2016-2017, Luke Sjulson started *TSToolbox2*, still in Matlab and which i
 In 2018, Francesco started neuroseries, a Python package built on Pandas. It was quickly adopted in Adrien's lab, especially by Guillaume Viejo, a postdoc in the lab. Gradually, the majority of the lab was using it and new functions were constantly added.
 In 2021, Guillaume and other trainees in Adrien's lab decided to fork from neuroseries and started *pynapple*. The core of pynapple is largely built upon neuroseries. Some of the original changes to TSToolbox made by Luke were included in this package, especially the *time_support* property of all ts/tsd objects.
 
+0.6.6 (Soon)
+------------------
+
+- Full lazy-loading for NWB file.
+- Parameter `load_array` for time series can prevent loading zarr array
+- Function to merge a list of `TsGroup`
```

Comment:
```suggestion
- Methods to merge multiple `TsGroup`
```
This would include the static method for merging a bunch of ts group as well as the merge method